src/main/java/seedu/address/logic/commands/listcommand/ListCommand.java

```diff
@@ -2,48 +2,66 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.Map;
+
+import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.entity.Entity;
+import seedu.address.model.entity.Participant;
+import seedu.address.model.entity.Team;
 
 /**
  * Lists every {@link Entity} (i.e. mentor, participant, team) in Alfred.
  */
 public class ListCommand extends Command {
 
-    /* Possible Fields */
+    public static final String MESSAGE_SUCCESS = "Listed all teams";
 
     public static final String COMMAND_TYPE = "list";
```

Comment:
Command_Type is basically the command word for the parser right? Could you just name it COMMAND_WORD instead to make it consistent with the rest of the commands.

Reply:
Oh yea my bad. Will change